static/js/common.js

```diff
@@ -62,6 +62,21 @@ exports.password_quality = function (password, bar, password_field) {
     return acceptable;
 };
 
+exports.password_warning = function (password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    if (password_field) {
+        var min_length = password_field.data('minLength') || min_length;
```

Comment:
we should remove the || min_length here. Also, in javascript, does the var have to be defined outside the if block?